projects/docs/docs/commands/cache.md

```diff
@@ -0,0 +1,54 @@
+---
+title: tuist cache
+slug: '/commands/cache'
+description: "Learn how to use Tuist's cache command to generate binary artifacts for your targets."
+---
+
+Target caching is one of Tuist's distictive features.
+It creates a binary artifact of your targets, which can later be used in your project if your generation is focused on other targets.
+For more details of the overall caching workflow, you can read the [caching]](/building-at-scale/caching/) documentation.
+
+### Warm
+
+To generate the cache for all the target of your project, youn can simply run:
+
+```bash
+tuist cache warm
+```
+
+You can even select a subset of targets to be cached, by passing them as command arguments:
+
+```bash
+tuist cache warm TargetA TargetB
+```
+
+The cached artifacts are stored in Tuist cache at `~/.tuist/Cache/BuildCache/<target-hash>`.
+If a target is already present in cache, it will not be built again by the command.
```

Comment:
```suggestion
The cached artifacts are stored in the Tuist cache at `~/.tuist/Cache/BuildCache/<target-hash>`.
If a target is already present in the cache, it will not be built again by the command.
```